### ISSUE TITLE
Support for a field 'domain_unit' during media_upload

### ISSUE THREAD

**Issue by deepshikha-jha** [Cogniac/cogniac-sdk-csharp#6]
Support for an Optional `domain_unit` field (e.g. serial number) for set assignment grouping.

Media with the same domain_unit will always be assigned to the same training or validation set. Set this to avoid overfitting when you have multiple images of the same thing or almost the same thing.

**Comment by deepshikha-jha** [comment]
yes this will be a string value. WE did archive some old tenants, let me tell you which one you can sue for testing

**Comment by alkadi** [comment]
Thanks.

**Comment by deepshikha-jha** [comment]
Has this been done yet. If not please concentrate on releasing this feature asap.

**Comment by alkadi** [comment]
Ok, if this is an urgent feature, I will make an SDK release just for this update.

**Comment by alkadi** [comment]
This has been implemented in package 1.0.9, the Upload tool utility has also been updated to support the domain unit, it is now version 1.0.0.2
